splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/util/util.js

```diff
@@ -0,0 +1,25 @@
+let appData = null;
+let unifiedConfigs = null;
+
+export function setMetaInfo(data) {
+    appData = data;
+}
+
+export function getMetaInfo() {
+    return {
+        appData: appData
+    }
+}
+
+export function generateEndPointUrl(name) {
+    console.log("unifiedConfig: ", unifiedConfigs);
```

Comment:
Did we put this console.log intentionally ?

Reply:
Yes, It's just for reference to how to get unifiedConfigs data

Reply:
Are you sure because, you already have put example in `TestComponent.jsx`. and this is util file. So this will log everytime we call `generateEndPointUrl`.